Index: ../ScalaLearning/src/main/scala/JodaTime.scala
===================================================================
diff --git a/../ScalaLearning/src/main/scala/JodaTime.scala b/../ScalaLearning/src/main/scala/JodaTime.scala
deleted file mode 100644
--- a/../ScalaLearning/src/main/scala/JodaTime.scala	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
+++ /dev/null	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
@@ -1,3 +0,0 @@
-object JodaTime {
-
-}
Index: ../ScalaLearning/pom.xml
===================================================================
diff --git a/../ScalaLearning/pom.xml b/../ScalaLearning/pom.xml
deleted file mode 100644
--- a/../ScalaLearning/pom.xml	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
+++ /dev/null	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>groupId</groupId>
-    <artifactId>ScalaLearning</artifactId>
-    <version>1.0-SNAPSHOT</version>
-
-    <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
-    </properties>
-    
-</project>
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
+++ /dev/null	(revision 4c2093e14bcc357e9f8ea4ff50695d3cdd325bf1)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
